test/tsickle_test.ts

```diff
@@ -200,11 +200,24 @@ function runGoldenTests(useSeparatePasses: boolean) {
       compareAgainstGolden(allExterns, test.externsPath);
 
       // Run tsickled TypeScript through TypeScript compiler
-      // and compare against goldens.
+      // and compare emit or diagnostics against goldens.
```

Comment:
Why include diagnostics here?  It seems it affects no tests?  If we want this, we should have a test that exhibits it.